examples/todos/package.json

```diff
@@ -31,9 +31,10 @@
     "babel-preset-react-hmre": "^1.1.1",
     "babel-register": "^6.3.13",
     "cross-env": "^1.0.7",
+    "enzyme": "^2.0.0",
     "expect": "^1.8.0",
     "express": "^4.13.3",
-    "jsdom": "^5.6.1",
+    "jsdom": "^8.1.0",
```

Comment:
Yeah, we can safely remove ```jsdom``` now. Good catch, thanks. I've updated the PR with 0baa696 to reflect this.